src/macros.hpp

```diff
@@ -77,4 +77,20 @@ struct StaticNextPow2 {
   enum { value = StaticNextPow2Helper<8 * sizeof(size_t) - 1, N>::value };
 };
 
+#if !defined(CASS_STATIC)
+#  if (defined(WIN32) || defined(_WIN32))
+#    if defined(CASS_BUILDING)
+#      define CASS_IMPL_EXPORT __declspec(dllexport)
+#    else
+#      define CASS_IMPL_EXPORT __declspec(dllexport)
```

Comment:
I'm pretty sure the `#else` should be `__declspec(dllimport)`. This is an existing typo. Let me do some research.